tests/test_packages/test_connections/test_p2p_libp2p/test_aea_cli.py

```diff
@@ -68,7 +68,7 @@ def teardown_class(cls):
         """Tear down the test"""
         cls.terminate_agents()
 
-        AEATestCaseEmpty.teardown_class()
+        super(TestP2PLibp2pConnectionAEARunningDefaultConfigNode, cls).teardown_class()
```

Comment:
why is this needed?

Reply:
incorrect call for classmethod.

AEATestCaseEmpty.teardown_class() - takes effect on AEATestCaseEmpty
not on current class instance.